core/paddlefl_mpc/mpc_protocol/aby3_operators_impl/elementwise_add_op.h

```diff
@@ -0,0 +1,137 @@
+/* Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+// Description: implementations of elementwise_add_op according to ABY3 protocol
+
+#pragma once
+
+#include "paddle/fluid/framework/tensor.h"
+#include "core/paddlefl_mpc/mpc_protocol/aby3_operators_impl/common.h"
+#include "core/paddlefl_mpc/operators/math/elementwise_op_function.h"
+#include "core/paddlefl_mpc/mpc_protocol/context_holder.h"
+
+namespace paddle {
+namespace operators {
+namespace aby3impl {
+
+using namespace paddle::operators::math;
+using namespace paddle::mpc;
+using CPUDeviceContext = paddle::platform::CPUDeviceContext;
+
+static void add_impl(const Tensor *lhs, const Tensor *rhs, Tensor *out, int axis) {
+
+    PADDLE_ENFORCE(lhs->dims()[0] == 2, "The first dimension of input x should be equal to 2.");
```

Comment:
better make it clear that the requirement is for ABY3 in the message

Reply:
done